conf/modules.config

```diff
@@ -599,23 +609,24 @@ if (!params.skip_alignment && params.aligner == 'star_salmon') {
             ]
         }
 
-        withName: '.*:QUANTIFY_STAR_SALMON:SALMON_TX2GENE' {
+        withName: '.*:QUANTIFY_STAR_SALMON:TX2GENE' {
             publishDir = [
                 path: { "${params.outdir}/${params.aligner}" },
                 mode: params.publish_dir_mode,
                 saveAs: { filename -> filename.equals('versions.yml') ? null : filename }
             ]
         }
 
-        withName: '.*:QUANTIFY_STAR_SALMON:SALMON_TXIMPORT' {
+        withName: '.*:QUANTIFY_STAR_SALMON:TXIMPORT' {
+            ext.prefix = { "${quant_type}.merged" }
```

Comment:
Where is the variable `quant_type` defined in this file?

Reply:
It a [val input](https://github.com/nf-core/rnaseq/blob/58bbc278ad3fd39bc0b28d63caec07ac8d83b162/modules/local/tximport/main.nf#L12) now. Per the nf-core guidelines mandatory value args go via channels.